packages/terra-i18n/src/I18nProvider.jsx

```diff
@@ -18,13 +18,26 @@ const propTypes = {
   messages: PropTypes.object.isRequired,
 };
 
-const I18nProvider = props => (
-  <IntlProvider locale={props.locale} key={props.locale} messages={props.messages}>
-    <div>
-      {props.children}
-    </div>
-  </IntlProvider>
-);
+const I18nProvider = (props) => {
+  const {
+    locale,
+    messages,
+    ...customProps
+  } = props;
+
+  return (
+    <IntlProvider
+      {...customProps}
+      locale={props.locale}
+      key={props.locale}
+      messages={props.messages}
```

Comment:
Nice catch: fix the props usage in this commit: https://github.com/cerner/terra-core/pull/219/commits/af32ffbfa09fda6d00b31cac83a5872477459641#diff-65d50c29354b491f8aa1ed537bde19baR23